datashuttle/utils/folders.py

```diff
@@ -89,12 +94,16 @@ def create_folder_trees(
             create_folders(ses_path, log)
 
             if datatype_passed:
-                datatype_paths = make_datatype_folders(
-                    cfg, datatype, ses_path, "ses", log=log
+                make_datatype_folders(
+                    cfg,
+                    datatype,
+                    ses_path,
+                    "ses",
+                    save_paths=all_paths,
+                    log=log,
                 )
-                all_paths.extend(datatype_paths)
             else:
-                all_paths.append(ses_path)
+                all_paths["ses"].append(sub_path)
```

Comment:
Quite sure you intended to use `ses_path` here.
```suggestion
                all_paths["ses"].append(ses_path)
```

Reply:
I guess this wasn't caught because this particular case (splitting output by sub/ses) is not tested (unless I missed it).